powerpacks.go

```diff
@@ -123,6 +126,40 @@ type NumberRemoveParams struct {
 	Unrent bool `json:"unrent,omitempty"`
 }
 
+type AddNumberOptions struct {
```

Comment:
All our types are defined under the `plivo` package. Please consider prefixing these types with `Powerpack` to prevent name ambiguity in the future with other features and products.